README.md

```diff
@@ -41,6 +41,28 @@
 
 如果感觉本书对你有所帮助，请点个 Star :star: 支持一下，谢谢！
 
+### 本地部署
```

Comment:
Please move the tutorial to `docs/chapter_preface/contribution.md`. Docker is not for most readers.